packages/web-components/src/components/va-summary-box/va-summary-box.tsx

```diff
@@ -30,10 +26,7 @@ export class VaSummaryBox {
   }
 
   componentDidLoad() {
-    if (!this.uswds) {
-      return
-    }
-    // add uswds classes
+       // add uswds classes
```

Comment:
Not sure if this comment is still needed?

Reply:
I felt like this was still relevant as it was describing what was happening in the function. I do think that the code here could be re-written now that v3 is the only option.